addons/payment_authorize/static/src/js/payment_form.js

```diff
@@ -0,0 +1,145 @@
+
+var _t = core._t;
+
+PaymentForm.include({
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * called when clicking on pay now or add payment event to create token for credit card/debit card.
+     *
+     * @private
+     * @param {Event} ev
+     * @param {DOMElement} checkedRadio
+     * @param {Boolean} addPmEvent
+     */
+    _createAuthorizeToken: function (ev, $checkedRadio, addPmEvent) {
+        var self = this;
+        var acquirerID = this.getAcquirerIdFromRadio($checkedRadio);
+        var acquirerForm = this.$('#o_payment_add_token_acq_' + acquirerID);
+        var inputsForm = $('input', acquirerForm);
+        var formData = self.getFormData(inputsForm);
+        if (this.options.partnerId === undefined) {
+            console.warn('payment_form: unset partner_id when adding new token; things could go wrong');
+        }
+        var AcceptJs = false;
+        if (formData.acquirer_environment === 'prod') {
+            AcceptJs = '//js.authorize.net/v3/AcceptUI.js';
+        } else {
+            AcceptJs = '//jstest.authorize.net/v3/AcceptUI.js';
+        }
+
+        window.responseHandler = function (response) {
+            _.extend(formData, response);
+
+            if (response.messages.resultCode === "Error") {
+                var errorMessage = "";
+                _.each(response.messages.message, function (message) {
+                    errorMessage += message.code + ": " + message.text;
+                })
+                acquirerForm.removeClass('d-none');
+                return self.displayError(_t('Server Error'), errorMessage);
+
+            }
+
+            self._rpc({
+                route: formData.data_set,
+                params: formData
+            }).then (function (data) {
+                if (addPmEvent) {
+                    if (formData.return_url) {
+                        window.location = formData.return_url;
+                    } else {
+                        window.location.reload();
+                    }
+                } else {
+                    $checkedRadio.val(data.id);
+                    self.el.submit();
+                }
+            }).guardedCatch(function (error) {
+                // if the rpc fails, pretty obvious
+                acquirerForm.removeClass('d-none');
+                self.enableButton(ev.target);
+                self.displayError(
+                    _t('Server Error'),
+                    _t("We are not able to add your payment method at the moment.") +
+                        error.message.data.message
+                );
+            });
+        };
+
+        var params = {
+            'class': 'AcceptUI d-none',
+            'data-apiLoginID': formData.login_id,
+            'data-clientKey': formData.client_key,
+            'data-responseHandler': 'responseHandler'
```

Comment:
also add
`data-billingAddressOptions: '{"show": false, "required": false}'`
to remove zip code